test/SlowTests/Authentication/AuthenticationDebugPackageInfoTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Collections.Generic;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Linq;
+using Raven.Client.ServerWide.Operations.Certificates;
+using Raven.Server.Documents.Handlers.Debugging;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Authentication
+{
+    public class AuthenticationDebugPackageTests : RavenTestBase
```

Comment:
I'm afraid that those tests might affect other running tests, please skip them

Reply:
I ran them on a Linux machine and the test fails because a Linux-only route errors (this issue https://issues.hibernatingrhinos.com/issue/RavenDB-17427). Could this be the issue instead of them interfering with other tests?

Reply:
Some of the endpoints that you are hitting by requesting a debug package are performing process-wide operations e.g. gather stacktraces, gather logs, etc
This is why it concerns me.

Reply:
Gathering log is an issue? I don't think we do anything with those anywhere in the tests.
The stackstrace - certainly, because we may pause the process and cause timeouts.

Reply:
Collecting stack traces is explicitly excluded in the tests so that's great. I'm a bit concerned as well but in general I'd like to have those tests.